sagemaker-training-compiler/huggingface/pytorch_single_gpu_single_node/albert-base-v2/albert-base-v2.ipynb

```diff
@@ -42,7 +42,7 @@
     "\n",
```

Comment:
<p>Do we need to install torch 1.12 instead of torch 1.13.1?</p><p>Also it seems that the torch installation is repeated twice.</p><p>And I think we simply don't need to install torch in this notebook, it's not imported anywhere. It's only required to be installed in the container.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3812/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Yes. You are right, we don't need to install torch here. I will remove this for all of three notebooks.